src/library/scala/StringContext.scala

```diff
@@ -10,6 +10,7 @@ package scala
 
 import java.lang.{ StringBuilder => JLSBuilder }
 import scala.annotation.tailrec
+import scala.language.experimental.macros
```

Comment:
not sure why this was needed but the compiler complained. 
def f was already a macro in this file previously